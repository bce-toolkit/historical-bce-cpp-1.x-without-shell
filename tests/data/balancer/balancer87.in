XeF4+CF3CFCF2=CF3CF2CF3+Xe