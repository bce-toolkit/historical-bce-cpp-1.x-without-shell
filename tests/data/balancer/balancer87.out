XeF4+2CF3CFCF2=2CF3CF2CF3+Xe